app/views/coverage/_filter.html.erb

```diff
@@ -3,9 +3,9 @@
     <div class="col-md-6 js-dynamic-cities js-only-available">
       <div class="form-group">
         <label>Required features:</label><br>
-        <% resource_klass::FEATURES.each do |val, human| %>
+        <% { voice_in: 'Voice IN', t38: 'T.38 Fax', sms_in: 'SMS IN' }.each do |val, human| %>
           <label class="checkbox-inline">
-            <%= check_box_tag 'q[features][]', val, filters['features']&.include?(val) %>
+            <%= check_box_tag 'q[features][]', val, filters['features']&.include?(val.to_s) %>
```

Comment:
why `to_s` is needed?

Reply:
@gigorok because val is symbol

Reply:
@gigorok 
I can define hash like this to avoid .to_s method

```
{ "voice_in" => 'Voice IN', "t38" => 'T.38 Fax', "sms_in" => 'SMS IN' }
```

Reply:
please change to string keys